dbsnap_verify/datadog_output.py

```diff
@@ -0,0 +1,65 @@
+import time
+
+# https://docs.datadoghq.com/integrations/amazon_lambda/#lambda-metrics
+DATADOG_METRIC_TYPES = {"count", "gauge", "histogram", "check"}
+
+# https://github.com/DataDog/datadogpy/blob/master/datadog/api/constants.py
+class CheckStatus(object):
```

Comment:
Since you only use this as a dict in this code, should probably just make it a dict.

Reply:
I copied / pasta'd the structure from here: https://github.com/DataDog/datadogpy/blob/master/datadog/api/constants.py 

I kept the structure the same just in case the upstream changes it will be easier to patch.